python/lsst/afw/geom/transform/transform.cc

```diff
@@ -81,7 +92,14 @@ void declareTransform(py::module &mod, std::string const &fromName, std::string
     cls.def("tranInverse", (FromArray (Class::*)(ToArray const &) const) & Class::tranInverse, "array"_a);
     cls.def("tranInverse", (FromPoint (Class::*)(ToPoint const &) const) & Class::tranInverse, "point"_a);
     cls.def("getInverse", &Class::getInverse);
+    // Need some extra handling of ndarray return type in Python
```

Comment:
This is helpful but specifics would be appreciated, e.g. "Need some extra handling of ndarray return type in Python to avoid dimensions of length 1 from being deleted, leaving a 1-dimensional array or scalar"

(if a 1 x 1 matrix is compressed to numpy scalar? if not, delete the last two words)